nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/QueryDatabaseTable.java

```diff
@@ -127,6 +129,7 @@
             .defaultValue("0")
             .required(true)
             .addValidator(StandardValidators.NON_NEGATIVE_INTEGER_VALIDATOR)
+            .expressionLanguageSupported(true)
             .build();
 
     public QueryDatabaseTable() {
```

Comment:
This is just a question out of curiosity, QueryDatabaseTable supports dynamic properties `Initial Max Value`, but GenerateTableFetch doesn't. As my understanding, QueryDatabaseTable and GenerateTableFetch are identical except the former execute SQL query while the latter generate flow files containing SQL. Initial Max Value dynamic property would be useful for GenerateTableFetch, too.

Also wondering how it will work if table name is specified by EL.